templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.secret.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sd-secret
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/service-account.name: sdsecret
+type: Opaque
+data:
+{{ (.Files.Glob "key/*").AsSecrets | indent 2 }}
```

Comment:
is key the folder the secrets reside ? maybe rename it as `secrets`
Is this supposed to hold one secrte per file ? How is the secret name handled ?